components/filter_adult/src/adult_set.rs

```diff
@@ -0,0 +1,8767 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+pub static ADULT_SET: [[u8; 16]; 2920] = [
```

Comment:
this seems, um, alot :) I assume it's generated - should we describe how it is generated?

(Also not sure it makes sense here, but https://doc.rust-lang.org/std/macro.include_str.html might be useful as an alternative - this would mean the Rust code ends up getting a very long string to parse at runtime, but the format of the string therefore becomes more flexibile (eg, you could imagine each line has a single base64 entry or similar)

Reply:
Yes, I can turn this into adult_set.bin, that's no problem.